bentoml/_internal/io_descriptors/image.py

```diff
@@ -116,16 +117,12 @@ async def from_http_request(self, request: Request) -> "PIL.Image":
     async def to_http_response(
         self, obj: t.Union["np.ndarray", "PIL.Image.Image"]
     ) -> Response:
-        if isinstance(obj, np.ndarray):
-            pass
-        elif isinstance(obj, PIL.Image.Image):
-            obj = np.array(obj)
-        else:
+        if not isinstance(obj, [np.ndarray, PIL.Image.Image]):
             raise InvalidArgument(
-                f"Unsupported Image type received: {type(obj)}, `{self.__class__.__name__}`"
-                " supports only `np.ndarray` and `PIL.Image`"
+                f"Unsupported Image type received: {type(obj)}, `{self.__class__.__name__}` supports only `np.ndarray` and `PIL.Image`"
             )
+        image = PIL.Image.fromarray(obj) if isinstance(obj, np.ndarray) else obj
 
         ret = io.BytesIO()
-        imageio.imsave(ret, im=t.cast(t.Type["np.ndarray"], obj), format=self._ext)
+        image.save(ret, self._ext)
```

Comment:
this is causing an error `KeyError: 'JPG'`, see https://github.com/bentoml/BentoML/runs/4151602877?check_suite_focus=true#logs

fixed it here: https://github.com/bentoml/BentoML/pull/1979/commits/7cf5ba4257e79933ca34a6038d80849b46235e4c